src/test/java/be/acara/events/controller/EventControllerTest.java

```diff
@@ -44,6 +45,29 @@ void findById() throws Exception {
                 .andExpect(jsonPath("$.category").value(Category.MUSIC.name()))
                 .andExpect(jsonPath("$.price").value("20.0"));
     }
+
+    @Test
+    void findAll() throws Exception {
+        Long idOne = 1L;
+        Long idTwo = 2L;
+        mockMvc.perform(get("/api/events"))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.eventList", hasSize(2)))
+                .andExpect(jsonPath("$.eventList[0].id").value(idOne))
+                .andExpect(jsonPath("$.eventList[0].eventDate").value(LocalDateTime.of(2020,12,20,20,30,54).toString()))
+                .andExpect(jsonPath("$.eventList[0].description").value("test description"))
+                .andExpect(jsonPath("$.eventList[0].image").value(compareBase64Image()))
+                .andExpect(jsonPath("$.eventList[0].location").value("genk"))
+                .andExpect(jsonPath("$.eventList[0].category").value(Category.MUSIC.name()))
+                .andExpect(jsonPath("$.eventList[0].price").value("20.0"))
+                .andExpect(jsonPath("$.eventList[1].id").value(idTwo))
+                .andExpect(jsonPath("$.eventList[1].eventDate").value(LocalDateTime.of(2020,11,21,21,31,55).toString()))
+                .andExpect(jsonPath("$.eventList[1].description").value("test description2"))
+                .andExpect(jsonPath("$.eventList[1].image").value(compareBase64Image()))
+                .andExpect(jsonPath("$.eventList[1].location").value("hasselt"))
+                .andExpect(jsonPath("$.eventList[1].category").value(Category.MUSIC.name()))
+                .andExpect(jsonPath("$.eventList[1].price").value("21.0"));
+    }
```

Comment:
This test fails for me. It appears that the expected values are swapped.

Reply:
Edited the test. Implemented sorting and forgot to update the test..